services/QuillLMS/engines/evidence/lib/evidence/evidence/automl_check.rb

```diff
@@ -24,27 +28,45 @@ def feedback_object
           category: ''
         }
       end
+
+      api = {
+        confidence: @confidence_score
+      }
+
+      if matched_rule.rule_type == Rule::TYPE_LOW_CONFIDENCE
+        api[:original_rule_uid] = matched_automl_rule&.uid
+        api[:original_rule_name] = matched_automl_rule&.name
+      end
+
       {
         feedback: feedback.text,
-        feedback_type: Rule::TYPE_AUTOML,
+        feedback_type: matched_rule.rule_type,
         optimal: matched_rule.optimal,
         entry: @entry,
         concept_uid: matched_rule&.concept_uid || '',
         rule_uid: matched_rule&.uid || '',
         hint: matched_rule&.hint,
         highlight: highlight,
-        api: {
-          confidence: @confidence_score
-        }
+        api: api
       }
     end
     # rubocop:enable Metrics/CyclomaticComplexity
 
-    private def matched_rule
-      @matched_rule ||= fetch_matched_rule
+    private def matched_low_confidence_rule
+      return unless matched_automl_rule && !matched_automl_rule.optimal && @confidence_score < LOW_CONFIDENCE_THRESHOLD
+
+      @matched_low_confidence_rule ||= fetch_matched_low_confidence_rule
```

Comment:
Looks like this instance var only exists for testing. It would be cleaner to stub the method `matched_low_confidence_rule` and just return a value on this line, rather than making an instance variable.